package.json

```diff
@@ -101,6 +101,7 @@
     "lodash.flatten": "^4.2.0",
     "lodash.isequal": "^4.1.1",
     "lodash.isobject": "^3.0.2",
-    "object-assign": "^4.0.1"
+    "object-assign": "^4.0.1",
+    "recompose": "^0.20.2"
```

Comment:
Is this a big dependency? Doesn't redux also have a compose function we could use? Or are they different?